.github/workflows/dev-backup.yml

```diff
@@ -4,10 +4,28 @@ on:
     - cron: '0 7 * * MON-FRI' # At 07:00 UTC on every day-of-week from Monday through Friday.
   workflow_dispatch:
 jobs:
+  setup:
```

Comment:
so does this mean that every step that uses the ECR image needs to have a setup? and the purpose of the setup isn't just to output the image path but to log docker into ECR or something like that, right?

do we need a step to output the image path btw - like can we not just hardcode it like we did with gitlab or dockerhub?

Reply:
The images in gitlab are hardcoded using environment variables. So it's configurable in 1 place.

We tried using secrets in GitHub i.e. 
```
    container:
      image: ${{ secrets.IMAGE_SQUIDEX_UTILS }}
```
But get get:

```
The workflow is not valid. .github/workflows/dev-backup.yml (Line: 10, Col: 14): Unrecognized named-value: 'secrets'. Located at position 1 within expression
```

The setup step was an attempt to avoid pasting the location of the images in multiple places. Downside, it will be needed in each step that uses it.

Reply:
Instead of requesting the account id via the AWS credentials, we're now hardcoding the account id in the action.

This reduces the runtime from 15 to 8 seconds.